inf-clojure.el

```diff
@@ -940,6 +940,25 @@ prefix argument PROMPT-FOR-SYMBOL, it prompts for a symbol name."
                (inf-clojure-symbol-at-point))))
     (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-source-form) var))))
 
+(defvar inf-clojure--dump-file-name ".inf-clojure.dump"
```

Comment:
The more Emacsy way would be to dump debug output to some buffer, not a file (e.g. cider's message's buffer). I'd also add some commands like `inf-clojure-toggle-debug` (or something like this) and use `debug` instead of `dump` as the terminology for this feature.